src/brod_supervisor.erl

```diff
@@ -1382,9 +1319,93 @@ init_state1(SupName, {Strategy, MaxIntensity, Period}, Mod, Args) ->
     period = Period,
     module = Mod,
     args = Args}};
-init_state1(_SupName, Type, _, _) ->
+init_state(_SupName, Type, _, _) ->
     {invalid_type, Type}.
 
+init_children(State, StartSpec) ->
```

Comment:
could you move this section back to the old place ?
this would make it more difficult to re-base or merge newer releases from otp.